packages/volto/src/server.jsx

```diff
@@ -69,10 +72,26 @@ const server = express()
   })
   .use(cookiesMiddleware());
 
+if (process.env.RAZZLE_PREFIX_PATH) {
+  server.use(
+    process.env.RAZZLE_PREFIX_PATH,
+    express.static(
+      process.env.BUILD_DIR
+        ? path.join(process.env.BUILD_DIR, 'public')
+        : process.env.RAZZLE_PUBLIC_DIR,
```

Comment:
Not only the static file middleware needs to be prefixed but other middlewares as well. And the `devProxy` and `ok` middlewares cannot be prefixed.

And as it is, it is not considering overwriting Volto files in the `public` folder of the addon root.